frontend/src/components/maps/leaflet/PointClusterer.scss

```diff
@@ -0,0 +1,41 @@
+@import 'fonts.scss';
+
+.marker-cluster-small {
+  background-color: rgba(181, 226, 140, 0.6);
+}
```

Comment:
I copied the CSS cluster styles from their examples. The default behaviour is to use different colors based on the size of the clusters; ie green for small (< 100 points), orange for medium (< 1000) and red-ish for large (> 1000)

The default styling can be changed. I'm just putting it up for UX review and can be tweaked if needed